### DISCUSSION TITLE
Rope 1.11.0 released

### DISCUSSION THREAD

**Discussion by lieryan** [python-rope/rope#725] [Announcements]
Date: 2023-11-06

- #710, #561 Implement `except*` syntax (@lieryan)
- #711 allow building documentation without having rope module installed (@kloczek)
- #719 Allows the in-memory db to be shared across threads (@tkrabel)
- #720 create one sqlite3.Connection per thread using a thread local (@tkrabel)
- #715 change AutoImport's `get_modules` to be case sensitive (@bagel897)

**Comment by kloczek** [comment]
Thank you 👍
